src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ResourceServerConfiguration.kt

```diff
@@ -24,7 +24,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
       .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
       .and().csrf().disable()
       .authorizeRequests {
-        it.antMatchers("/health/**", "/info", "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html").permitAll()
+        it.antMatchers("/health/**", "/meta/schema", "/info", "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html").permitAll()
```

Comment:
do we want this to be accessible to the public?

Reply:
I think so. It's the schema of the data, without the data itself. Should we restrict it?

Reply:
i guess our source code is public so it makes no difference 👍